<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRolesForm</class>
 <widget class="QDialog" name="frmRolesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo rol</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/assets/users.png</normaloff>:/assets/users.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>506</width>
           <height>224</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="formContainer">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="lblRol">
                <property name="text">
                 <string>Rol:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtRol"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>txtRol</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>frmRolesForm</sender>
   <signal>rejected()</signal>
   <receiver>frmRolesForm</receiver>
   <slot>closeParent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>frmRolesForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAceptar</sender>
   <signal>clicked()</signal>
   <receiver>frmRolesForm</receiver>
   <slot>SaveRol()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeParent()</slot>
  <slot>SaveRol()</slot>
 </slots>
</ui>
